src/meta/src/rpc/metrics.rs

```diff
@@ -67,20 +62,18 @@ pub struct MetaMetrics {
 impl MetaMetrics {
     pub fn new() -> Self {
         let registry = prometheus::Registry::new();
-        let buckets = DEFAULT_BUCKETS;
         let opts = histogram_opts!(
             "meta_grpc_duration_seconds",
             "gRPC latency of meta services",
-            buckets.iter().map(|x| *x * 0.1).collect_vec()
+            exponential_buckets(0.0001, 2.0, 16).unwrap() // max 6s
         );
         let grpc_latency =
             register_histogram_vec_with_registry!(opts, &["path"], registry).unwrap();
 
-        let buckets = BARRIER_BUCKETS;
         let opts = histogram_opts!(
             "meta_barrier_duration_seconds",
-            "barrier latency ",
-            buckets.to_vec()
+            "barrier latency",
+            exponential_buckets(0.000005, 1.6, 36).unwrap() // max 115s
```

Comment:
Is the start latency  5us? It seems too small..... Our barrier can not run so fast.....I suggest 100ms instead.